pkg/operator/vsphere_check.go

```diff
@@ -340,23 +364,24 @@ func getVM(checkContext *check.CheckContext, node *v1.Node) (*mo.VirtualMachine,
 	return &o, nil
 }
 
-func parseConfig(data string) (*vsphere.VSphereConfig, error) {
-	var cfg vsphere.VSphereConfig
-	err := gcfg.ReadStringInto(&cfg, data)
+func parseConfig(data string) (*util.VSphereConfig, error) {
+	cfg := &util.VSphereConfig{}
+	err := cfg.LoadConfig(data)
+	//cfg, err := vspherecfg.ReadConfig([]byte(data))
```

Comment:
this line (`//cfg, ...`) looks as leftover, let's remove it.

Reply:
I'll remove this.  Thanks!